tests/system_tests.pm

```diff
@@ -30,8 +30,6 @@ sub run {
     curl_via_netvm;
 
     assert_script_run("sudo cat /root/extra-files/convert_junit.py >convert_junit.py");
-    # until https://github.com/nose-devs/nose2/pull/412 gets merged
-    assert_script_run("sudo patch /usr/lib/python3*/site-packages/nose2/plugins/junitxml.py /root/extra-files/nose2-junit-xml-log-skip-reason.patch");
```

Comment:
I believe this patch is still needed for running R4.0 tests, so please make it conditional.

Reply:
yes it fall into a wormhole when we discussed it a moment ago and it disappeared of my memory :)